code/csharp/main-workshop/src/CdkWorkshop/CdkWorkshop.csproj

```diff
@@ -2,19 +2,18 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <!-- CDK Construct Library dependencies -->
-    <PackageReference Include="Amazon.CDK" Version="1.17.1-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.APIGateway" Version="1.17.1-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.DynamoDB" Version="1.17.1-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.Lambda" Version="1.17.1-devpreview" />
+    <PackageReference Include="Amazon.CDK" Version="1.*" />
+    <PackageReference Include="Amazon.CDK.AWS.APIGateway" Version="1.*" />
+    <PackageReference Include="Amazon.CDK.AWS.DynamoDB" Version="1.*" />
+    <PackageReference Include="Amazon.CDK.AWS.Lambda" Version="1.*" />
 
-    <PackageReference Include="Eladb.DynamoTableViewer" Version="3.0.6" />
+    <PackageReference Include="Eladb.DynamoTableViewer" Version="[0.1.*,3)" />
```

Comment:
This version specification was required in order for Eladb.DynamoTableViewer to share compatible peer dependencies with the latest CDK packages.